jsbits/Scenario.js

```diff
@@ -115,10 +142,11 @@ function gm$updateProps(bArray, values) {
  * @param fc - FeatureCollection
  * @returns {[points:Feature,lines:Feature,surfaces:Feature,deletes:Number(geomID),errors:string,cmin,cmax,dims]}
  */
-function gm$smartProcessFeatureCollection(fc, defVec, maxGeomId) {
+
+function gm$smartProcessFeatureCollection(fc, coorSys, defVec, maxGeomId) {
```

Comment:
Instead of passing coorSys, you can just pass transform: bool to feature collection. The only type of transform we have WGS84->metric anyway.